server/core/sideloader.js

```diff
@@ -52,15 +52,17 @@ module.exports = {
                   strings: lcObj,
                   isRTL: locale.isRTL,
                   name: locale.name,
-                  nativeName: locale.nativeName
+                  nativeName: locale.nativeName,
+                  availability: locale.availability ?? 0
```

Comment:
Don't use the ?? operator as its not available under Node 12 which has to be supported. Use || instead

Reply:
Fix is pushed